src/components/Header/index.scss

```diff
@@ -1,23 +1,88 @@
 @import "../../styles/variables.scss";
 
 .Header {
-  margin-bottom: 1.45rem;
+  position: fixed;
+
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+
+  width: 100%;
+
+  padding: $size-base $size-xxlarge $size-base $size-xxlarge;
 
   background-color: $color-deep-blue;
 
-  h1 {
+  color: white;
+
+  h2 {
     margin: 0;
   }
 }
 
-.Header-inner {
-  max-width: 960px;
-  padding: 50px 20px 50px 20px;
-  margin: 0 auto;
+.Header-left {
+  justify-content: flex-start;
 }
 
 .Header-link {
   color: white !important;
 
   text-decoration: none !important;
 }
+
+.Header-logo {
+  font-size: $fontSize-medium;
+  font-family: $font-family-mono;
+  font-weight: $font-weight-bold;
+  vertical-align: text-bottom;
+  white-space: nowrap;
+}
+
+.Header-sections {
+  display: flex;
+  flex-direction: row;
+
+  a:not(:last-child) {
+    margin-right: $size-base;
+  }
+
+  h3 {
+    margin: 0;
```

Comment:
This should probably be fixed by a `reset.css`

Reply:
:+1: